spec/requests/reviewables_controller_spec.rb

```diff
@@ -254,6 +254,47 @@
       end
     end
 
+    context "#lightweight_index" do
+      it "returns JSON containing basic information of reviewables" do
+        reviewable1 = Fabricate(:reviewable)
+        reviewable2 = Fabricate(:reviewable, status: Reviewable.statuses[:approved])
+        get "/review/lightweight-list", xhr: true
```

Comment:
```suggestion
        get "/review/lightweight-list.json"
```

Is there a reason we need `xhr: true`?